view_all_set.php

```diff
@@ -374,11 +94,6 @@
 	$f_per_page = config_get( 'default_limit_view' );
```

Comment:
While testing, I got a `SYSTEM NOTICE: 'Undefined variable: f_per_page' in 'view_all_set.php'` at line 93